app/components/Search/ResourceEntry.jsx

```diff
@@ -52,7 +55,11 @@ class ServiceEntry extends Component {
           <div className="entry-details">
             <h4 className="entry-headline">{hit.name}</h4>
             <div className="entry-subhead">
-              <p>{`${hit.address ? hit.address.address_1 : 'No address found'} • ${timeInfo}`}</p>
+              <p>
+                { hit.address && hit.address.address_1 ? hit.address.address_1 : 'No address found' }
+                { hit.schedule ? ' • ' : null }
```

Comment:
Should this be `schedule` instead of `hit.schedule`? This would make this code more parallel to the conditional one line below.

Reply:
Yeah you're right, I thought this after I'd committed actually but didn't bother to go back and change. Patching now.